Pool/ScoreTheme.vb

```diff
@@ -73,89 +48,87 @@ Public Class ScoreTheme
-            With cBox
-                .BackColor = Color.Aqua
-            End With
-        Next
-    End Sub
-
-    Public Sub SetLabel(ByRef lbl As Label())
-        For Each label In lbl
-            With label
-                .ForeColor = Color.Lime
-                .Font = New Font("Gill Sans Ultra", 10, FontStyle.Bold)
-            End With
-        Next
-    End Sub
 
 #End Region
-    Public Function GetKey(ByVal values As String, ByVal col As Hashtable) As Integer
+
+    ''' <summary>
+    ''' Retrieve key ID for this value
+    ''' </summary>
+    ''' <param name="values"></param>
+    ''' <param name="col"></param>
+    ''' <returns></returns>
+    Public Shared Function GetKey(ByVal values As String, ByVal col As Hashtable) As Integer
         For Each items As DictionaryEntry In col
             If items.Value.Equals(values) Then
                 Return items.Key
             End If
         Next
         Return -1
     End Function
-    Public Sub FillBoxfromHT(ByRef cBox As ComboBox, ByVal allPlayersDT As Hashtable)
-        cBox.Items.Clear()
-        For Each player As DictionaryEntry In allPlayersDT
-            If Not player.Value.Equals("Error") Then
-                cBox.Items.Add(player.Value)
-            End If
-        Next
-    End Sub
-    Public Sub FillCBoxAll(ByVal allList As Hashtable, ByRef cBox As ComboBox, ByRef lblError As Label)
+
+    ''' <summary>
+    ''' Fill combox from hashtable
+    ''' </summary>
+    ''' <param name="cBox"></param>
+    ''' <param name="allDT"></param>
+    Public Shared Sub FillBoxfromHT(ByRef cBox As ComboBox, ByVal allDT As Hashtable)
         cBox.Items.Clear()
-        For Each name As DictionaryEntry In allList
-            If Screen = 0 AndAlso name.Value.ToString.Contains("Error") Then
-                Screen = -1
-                lblError.Text = "Error getting all players"
-                Exit For
-            Else
-                cBox.Items.Add(name.Value)
-                Screen = AppState.SelectPlayer
+        cBox.Items.Add("Choose")
+        For Each entry As DictionaryEntry In allDT
+            If Not entry.Value.Equals("Error") Then
+                cBox.Items.Add(entry.Value)
             End If
         Next
     End Sub
 
-    Public Function ValidateCBox(ByRef cBox As ComboBox) As Boolean
-        If cBox.SelectedItem IsNot Nothing AndAlso Not String.IsNullOrEmpty(cBox.SelectedItem.ToString) Then
+    'Public Sub FillCBoxAll(ByVal allList As Hashtable, ByRef cBox As ComboBox, ByRef lblError As Label)
+    '    cBox.Items.Clear()
+    '    For Each name As DictionaryEntry In allList
+    '        If Screen = 0 AndAlso name.Value.ToString.Contains("Error") Then
+    '            Screen = -1
+    '            lblError.Text = "Error getting all players"
+    '            Exit For
+    '        Else
+    '            cBox.Items.Add(name.Value)
+    '            Screen = AppState.SelectPlayer
+    '        End If
+    '    Next
+    'End Sub
```

Comment:
See if we are really not using this, if not then remove it with this branch.

Reply:
This is not being used anymore, it has been updated to the new fill function and has been removed.